c2a_core_main.h

```diff
@@ -4,4 +4,11 @@
 void C2A_core_init(void);
 void C2A_core_main(void);
 
+// C2A の バージョンは Semantic Versioning を採用する
+// これらの番号は，リリース時に手動で合わせる
+// 詳細: Docs/General/release.md
+#define C2A_CORE_VER_MAJOR 3
+#define C2A_CORE_VER_MINOR 4
+#define C2A_CORE_VER_PATCH 0
```

Comment:
```suggestion
#define C2A_CORE_VER_MAJOR (3)
#define C2A_CORE_VER_MINOR (4)
#define C2A_CORE_VER_PATCH (0)
```

がいい気がしてきたので，これだけさっと直してしまおう．